src/Templates/src/templates/maui-lib/.template.config/template.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://json.schemastore.org/template",
+    "author": "Microsoft",
+    "classifications": [ "MAUI", "Android", "iOS", "macOS", "Mac Catalyst", "WinUI" ],
+    "identity": "Microsoft.Maui.MauiLib",
+    "groupIdentity": "Microsoft.Maui",
+    "name": ".NET MAUI Class Library",
+    "description": "A project for creating a .NET MAUI class library",
+    "shortName": "mauilib",
```

Comment:
Are we happy with `mauilib`? `maui-lib`? `maui-classlib`?

Reply:
I prefer shorter. `mauilib`